.github/CODEOWNERS

```diff
@@ -8,71 +8,69 @@
 * @williamfalcon @borda @tchaton @awaelchli @kaushikb11 @rohitgr7
 
 # CI/CD and configs
-/.github/       @borda @carmocca @akihironitta @tchaton
-/.circleci/     @borda @carmocca @akihironitta @tchaton
-/.azure/        @borda @carmocca @akihironitta @tchaton
-/dockers/       @borda @carmocca @akihironitta @tchaton
-*.yml           @borda @carmocca @akihironitta @tchaton
+/.github/       @borda @carmocca @akihironitta @otaj
+/.circleci/     @borda @carmocca @akihironitta @otaj
+/.azure/        @borda @carmocca @akihironitta @otaj
+/dockers/       @borda @carmocca @akihironitta @otaj
+*.yml           @borda @carmocca @akihironitta @otaj
```

Comment:
@otaj I welcome you to CI ownership :imp:

Reply:
Lovely :smiling_imp: